docs/Reward-Signals.md

```diff
@@ -135,7 +135,7 @@ discriminator is trained to better distinguish between demonstrations and agent
 In this way, while the agent gets better and better at mimicing the demonstrations, the
 discriminator keeps getting stricter and stricter and the agent must try harder to "fool" it.
 
-This approach, when compared to [Behavioral Cloning](Training-Behavioral-Cloning.md), requires
+This approach, when compared to Behavioral Cloning, requires
```

Comment:
Maybe link to the paper?

Reply:
Or maybe just shorten the paragraph to "This approach is especially effective when combined with an Extrinsic signal..."

Reply:
You're right, this paragraph was pretty awful... I revised it a bit